epub33/core/index.html

```diff
@@ -7850,178 +7850,178 @@ <h4>Content document dimensions</h4>
 				</section>
 			</section>
 
-			<section id="sec-reflowable-layouts">
-				<h3>Reflowable layouts</h3>
-
-				<p>Although control over the rendering of [=EPUB content documents=] to create <a
-						href="#sec-fixed-layouts">fixed layouts</a> is an obvious need not handled by other
-					technologies, there are also considerations for reflowable content that are unique to [=EPUB
-					publications=] (e.g., how to handle the flow of content in the [=viewport=]). This section defines
-					properties that allow [=EPUB creators=] to control presentation aspects of reflowable content.</p>
+		<section id="flow" data-epubcheck="true"
+			data-tests="https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L320,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L325,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L332,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L339">
 
-				<section id="flow" data-epubcheck="true"
-					data-tests="https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L320,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L325,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L332,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L339">
-					<h4>The <code>rendition:flow</code> property</h4>
+			<h3>Layout rendering control</h3>
```

Comment:
Was this change intentional? The h3 here is the same as the parent section's:

> 8. Layout rendering control
> ...
> 8.3 Layout rendering control

Reply:
I can't think of a really great title off the top of my head, but it seems like it should be more along the lines of "Layout-independent settings" (and maybe come after the fixed and reflowable sections?).

Beneath that it might help to keep a distinct subsection for rendition:flow in case there are other properties in the future, so a structure like:

> 8.4 Layout-independent settings
> 8.4.1 The rendition:flow property
> 8.4.1.1 Spine overrides

Reply:
I agree with the possible change in the section title (and I like the 'Layout-independent' alternative).